src/Serilog/Parameters/PropertyValueConverter.cs

```diff
@@ -121,11 +125,22 @@ LogEventPropertyValue CreatePropertyValue(object value, Destructuring destructur
                 return new ScalarValue(null);
 
             if (destructuring == Destructuring.Stringify)
-                return new ScalarValue(value.ToString());
+            {
+                return Stringify(value);
+            }
 
             var valueType = value.GetType();
             var limiter = new DepthLimiter(depth, _maximumDestructuringDepth, this);
 
+            if (destructuring == Destructuring.Destructure)
```

Comment:
Ah, I see - thanks, yes you're right. We should continue not to limit regular string logging.

Let's merge this one; I'm looking forward to moving onto collection size and so-on.